src/addons/activeproxy/activeproxy.h

```diff
@@ -165,6 +165,23 @@ class ActiveProxy : public Addon {
 
     void announcePeer() noexcept;
 
+    void resetValues() noexcept {
```

Comment:
It's reset the whole AcitveProxy internal state. So it's better to use the prototype define like - 
void reset() noexcept { xxxxx}